azuredevops/internal/utils/pipelineschecksextras/pipelineschecks_extras.go

```diff
@@ -0,0 +1,59 @@
+package pipelineschecksextras
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/google/uuid"
+	"github.com/microsoft/azure-devops-go-api/azuredevops"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/pipelineschecks"
+)
+
+var ResourceAreaId, _ = uuid.Parse("4a933897-0488-45af-bd82-6fd3ad33f46a")
+
+type Client interface {
+	// [Preview API] Get Check configuration by Id
+	GetCheckConfiguration(context.Context, pipelineschecks.GetCheckConfigurationArgs) (*pipelineschecks.CheckConfiguration, error)
+}
+
+type ClientImpl struct {
+	Client azuredevops.Client
+}
+
+func NewClient(ctx context.Context, connection *azuredevops.Connection) (Client, error) {
+	client, err := connection.GetClientByResourceAreaId(ctx, ResourceAreaId)
+	if err != nil {
+		return nil, err
+	}
+	return &ClientImpl{
+		Client: *client,
+	}, nil
+}
+
+// [Preview API] Get Check configuration by Id
+func (client *ClientImpl) GetCheckConfiguration(ctx context.Context, args pipelineschecks.GetCheckConfigurationArgs) (*pipelineschecks.CheckConfiguration, error) {
+	routeValues := make(map[string]string)
+	if args.Project == nil || *args.Project == "" {
+		return nil, &azuredevops.ArgumentNilOrEmptyError{ArgumentName: "args.Project"}
+	}
+	routeValues["project"] = *args.Project
+	if args.Id == nil {
+		return nil, &azuredevops.ArgumentNilError{ArgumentName: "args.Id"}
+	}
+	routeValues["id"] = strconv.Itoa(*args.Id)
+
+	queryParams := url.Values{}
+	queryParams.Add("$expand", "1")
```

Comment:
```suggestion
	queryParams.Add("$expand", "settings")
```
Though any other values works, `settings` has a more specific meaning.
Ref: https://learn.microsoft.com/en-us/rest/api/azure/devops/approvalsandchecks/check-configurations/get?view=azure-devops-rest-7.0&tabs=HTTP#checkconfigurationexpandparameter

Reply:
Interesting, didn't realize it was more like an enumeration than a boolean, applied suggestion.